gradle/dependencies.gradle

```diff
@@ -12,17 +12,17 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '87.0.5'
+      mapboxNavigatorVersion = '88.0.0'
   }
   println("Navigation Native version: " + mapboxNavigatorVersion)
 
   version = [
-      mapboxMapSdk              : '10.3.0-rc.1',
+      mapboxMapSdk              : '10.3.0',
       mapboxSdkServices         : '6.3.0-beta.1',
```

Comment:
We also need a stable release of mapbox-java for Nav SDK v2.3. We can do that for RC.2/final though.